### PR TITLE
fix: fix AdditionalApplicationsUploadedEventHandlerTest (FPLA-3307)

### PR DESCRIPTION
**Pull request by tomaszpowroznik**
https://tools.hmcts.net/jira/browse/FPLA-3307

static mocks should not be used as they keep state between tests execution. Sometimes it is enough to change order of test execution or run in multiple times to have unexpected behaviour. By default junit framework creates new instance of test class before running each test to keep them isolated. With static mocks this isolation is broken

### CODE REVIEW

File: service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/AdditionalApplicationsUploadedEventHandlerTest.java [hunk 1]
```diff
@@ -101,22 +101,24 @@
...
-
     private static final String EMAIL_REP_1 = "email-rep1@test.com";
     private static final String EMAIL_REP_2 = "email-rep2@test.com";
     private static final Set<String> EMAIL_REPS = new HashSet<>(Arrays.asList(EMAIL_REP_1, EMAIL_REP_2));
     private static final String DIGITAL_REP_1 = "digital-rep1@test.com";
     private static final String DIGITAL_REP_2 = "digital-rep2@test.com";
+    private static final String SUBJECT_LINE = "Lastname, SACCCCCCCC5676576567";
     private static final Set<String> DIGITAL_REPS = new HashSet<>(Arrays.asList(DIGITAL_REP_1, DIGITAL_REP_2));
     private static final List<Element<Other>> NO_RECIPIENTS = Collections.emptyList();
     private static final List<Element<Other>> SELECTED_OTHERS = List.of(element(mock(Other.class)));
```
Comment (lewisbirks) [inline service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/AdditionalApplicationsUploadedEventHandlerTest.java:119]: there is a static mock here, do we want to remove this one as well?

Comment (tomaszpowroznik) [reply service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/AdditionalApplicationsUploadedEventHandlerTest.java:119]: this is rather harmless, as there is not behaviour recorder for it nor verification.

**Comment by Rob-Doherty** [comment]
Running tests via nightly pipeline here: https://build.platform.hmcts.net/view/FPLA/job/HMCTS_Nightly_FPL/job/fpl-ccd-configuration/view/change-requests/job/PR-2983/2/

But the pipeline has progressed past that previous failure, so looking good
`11:39:32  10:39:31 AM PIT >> INFO : Created  531 mutation test units`

**Review by lewisbirks — APPROVED** [review]

**Comment by Rob-Doherty** [comment]
image

**Pull request merged**